### DISCUSSION TITLE
Custom keyboard rebinds in editor

### DISCUSSION THREAD

**Discussion by dec0dOS** [standardnotes/forum#3095] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?** yes

**Is your feature request related to a problem? Please describe.**
In SN Discord is often asked about the VIM keybinding for note-editing

**Describe the solution you'd like**
Some sort of additional plugin architecture, so you could extend the provided editors.

**Describe alternatives you've considered**
Provide popular keybinding for Super Editor

**Comment by effieeee** [comment]
Hi @dec0dOS, the **Code** note type has a **Vim mode** that you can toggle on the lower right for Vim functions!

![image](https://github.com/standardnotes/forum/assets/54989748/3985eb9e-c32f-4e5f-ad89-06f31503f5b1)
